packages/plugin-import-export/src/utilities/collectDisabledFieldPaths.ts

```diff
@@ -0,0 +1,43 @@
+import { fieldAffectsData } from 'payload/shared'
+
+export const collectDisabledFieldPaths = (fields: any[], path: string[] = []): string[] => {
```

Comment:
We need to use get in the habit of `traverseFields` from `payload`. We have so many recursive fields functions that if we ever add a new field type or change behavior of containers it will be a huge job to remember all the functions that need to be changed.